.github/workflows/regen-builds.yml

```diff
@@ -0,0 +1,20 @@
+name: Regen Builds
+on:
+  release:
+    types: [published, released]
+  workflow_run:
+    workflows: [Build]
+    types: [completed]
```

Comment:
Maybe a branch restriction for `master or \d_\d_X `(if a regex is allowed) or a check for the branch name might be good? If someone pushes a branch to the repo I think this would currently run for that branch?

Reply:
Fixed. You can't do regexes, but you can do glob patterns!